state/state.go

```diff
@@ -121,34 +121,49 @@ func (s *State) MemberDisplayName(guildID, userID discord.Snowflake) (string, er
 	return member.Nick, nil
 }
 
-func (s *State) AuthorColor(message *gateway.MessageCreateEvent) discord.Color {
+func (s *State) AuthorColor(message *gateway.MessageCreateEvent) (discord.Color, error) {
 	if !message.GuildID.Valid() {
-		return discord.DefaultMemberColor
+		return 0, nil
```

Comment:
Is DefaultMemberColor removed?

Reply:
It's still used in discord.MemberColor

Reply:
But this could return a 0 still, so why not make it return default?

Reply:
Let me fix that.